executor/common_kvm_amd64.h

```diff
@@ -800,3 +800,26 @@ static volatile long syz_kvm_setup_cpu(volatile long a0, volatile long a1, volat
 		return -1;
 	return 0;
 }
+
+#define FEATURE_INTEL 0x00000001
+#define FEATURE_INTEL_ECX_VMX (1 << 5)
+
+#define FEATURE_AMD 0x800000001
+#define FEATURE_AMD_ECX_SVM (1 << 2)
+
+static long cpu_feature_enabled(uint32_t function, uint32_t eax_bits, uint32_t ebx_bits, uint32_t ecx_bits, uint32_t edx_bits)
```

Comment:
This should go in the test file. Otherwise it will appear in syz-executor binary and in C reproducers.

Reply:
Got it. Thanks.